frontend/src/pages/Assignments/Assignment/index.tsx

```diff
@@ -95,6 +95,17 @@ const Assignment: React.FC = () => {
     );
   }, [selectedClassroom]);
 
+  useEffect(() => {
+    if (!selectedClassroom || !id) return;
+    getAssignmentTemplate(selectedClassroom.id, Number(id))
```

Comment:
Can we rename id to assignmentID please?

Reply:
fixed